lib/pluginmanager/util.rb

```diff
@@ -1,12 +1,14 @@
 require "rubygems/package"
 
 module LogStash::PluginManager
+
   # check for valid logstash plugin gem name & version or .gem file, logs errors to $stdout
   # uses Rubygems API and will remotely validated agains the current Gem.sources
   # @param plugin [String] plugin name or .gem file path
   # @param version [String] gem version requirement string
   # @return [Boolean] true if valid logstash plugin gem name & version or a .gem file
-  def self.logstash_plugin?(plugin, version = nil)
+  def self.logstash_plugin?(plugin, version = nil, options={})
```

Comment:
it would be nice to keep the method description comments up to date with the new options parameter.